### DISCUSSION TITLE
Loading dev version of the mod

### DISCUSSION THREAD

**Discussion by tsr488** [Aktanusa/CookieMonster#846] [General]
In the readme, under "Bugs and Suggestions," it states:

> You can also load the dev-version by using https://cookiemonsterteam.github.io/CookieMonster/dist/CookieMonsterDev.js to load the mod.

This is not actually true because that file is served from the `gh-pages` branch, so it doesn't have the latest changes from the dev branch. Perhaps this statement should be removed from the readme.

Thinking longer-term, I have two questions:
1. How can users easily load the dev version of the mod, other than copy pasting the JS directly into the console?
2. Is there any need for the /dist/ directory to contain a separate `CookieMonsterDev.js` file? It doesn't seem to serve much purpose.

**Comment by DanielNoord** [comment]
This is true indeed.
I wanted to set up automatically merging the dev-branch into the gh-pages branch but never got round to doing it.

The `CookieMonsterDev.js` file is there to allow hosting the dev-branch simultaneously with the live version. So, if the dev-branch contains an update we want to test extensively we can manually merge the dev-branch into the gh-pages branch and test it using the link.

The problem would thus be solved by setting up a workflow to automatically merge dev into gh-pages, but I currently don't have time to figure that out. Anybody who would like to try to set it up is very welcome to do so!

**Reply by tsr488** [reply]
I took a crack at it. https://github.com/CookieMonsterTeam/CookieMonster/pull/847

**Reply by DanielNoord** [reply]
With #847 merged, I'm closing this discussion
